### ISSUE TITLE
Similar implementation for your reference

### ISSUE THREAD

**Issue by Sunnydreamrain** [titu1994/Keras-IndRNN#2]
Hi,

I have uploaded my code of IndRNN to GitHub as well as some running examples using it.  
https://github.com/Sunnydreamrain/IndRNN_Theano_Lasagne  

Since there are a few items in your to-do list, I was thinking that maybe a similar code can assist your progress.

Thanks.

**Comment by titu1994** [comment]
Fantastic. I'll give it a look and try to incorporate batchnorm into it.
